docs/sender.rst

```diff
@@ -214,172 +282,227 @@ The transaction is automatically committed when the ``with`` block is exited.
+.. _sender_advanced:
 
-HTTP Basic Auth
-~~~~~~~~~~~~~~~
-
-* ``username`` - ``str``: Username for HTTP basic authentication.
-* ``password`` - ``str``: Password for HTTP basic authentication.
-
-HTTP Bearer Token
-~~~~~~~~~~~~~~~~~
-* ``token`` - ``str``: Bearer token for HTTP authentication.
-
-TLS
----
+Advanced Usage
+==============
 
-TLS in enabled by selecting the ``tcps`` or ``https`` protocol.
+Independent Buffers
+-------------------
 
-* ``tls_ca`` - The remote server's certificate authority verification mechamism.
+All examples so far have shown appending data to the sender's internal buffer.
 
-  * ``'webpki_roots'``: Use the
-    `webpki-roots <https://crates.io/crates/webpki-roots>`_ Rust crate to
-    recognize certificates.
+You can also create independent buffers and send them independently.
 
-  * ``'os_roots'``: Use the OS-provided certificate store.
+This is useful for more complex applications whishing to decouple the
+serialisation logic from the sending logic.
 
-  * ``'webpki_and_os_roots'``: Use both the
-    `webpki-roots <https://crates.io/crates/webpki-roots>`_ Rust crate and
-    the OS-provided certificate store to recognize certificates.
+Note that the sender's auto-flushing logic will not apply to independent
+buffers.
 
-  * ``pem_file``: Path to a PEM-encoded certificate authority file.
-    This is useful for testing with self-signed certificates.
+.. code-block:: python
 
-  The default is: ``'webpki_roots'``.
+    from questdb.ingress import Buffer, Sender, TimestampNanos
 
-* ``tls_roots`` - ``str``: Path to a PEM-encoded certificate authority file.
-  When used it defaults the ``tls_ca`` to ``'pem_file'``.
+    buf = Buffer()
+    buf.row(
+        'weather_sensor',
+        symbols={'id': 'toronto1'},
+        columns={'temperature': 23.5, 'humidity': 0.49},
+        at=TimestampNanos.now())
+    buf.row(
+        'weather_sensor',
+        symbols={'id': 'dubai2'},
+        columns={'temperature': 41.2, 'humidity': 0.34},
+        at=TimestampNanos.now())
 
-* ``tls_verify`` - ``'on'`` | ``'unsafe_off'``: Whether to verify the server's
-  certificate. This should only be used for testing as a last resort and never
-  used in production as it makes the connection vulnerable to man-in-the-middle
-  attacks.
-  
-  The default is: ``'on'``.
+    conf = 'http::addr=localhost:9000;'
+    with Sender.from_conf(conf) as sender:
+        sender.flush(buf, transaction=True)
 
-.. _configuring-sender-auto-flushing:
+The ``transaction`` parameter is optional and defaults to ``False``.
+When set to ``True``, the buffer is guaranteed to be committed as a single
+transaction, but must only contain rows for a single table.
 
-Auto-flushing
--------------
+Multiple Databases
+------------------
 
-The following paramers control the :ref:`sender-auto-flushing` behavior.
+Handling buffers explicitly is also useful when sending data to multiple
+databases via the `.flush(buf, clear=False)` option.
```

Comment:
There are sometimes double back ticks and sometimes single back ticks.

Dunno this templating language, so flagging in case it's inconsistent.

Reply:
A yes. That's a typo on my end! Thank you!

Single back ticks are for links (to URLs, classes, functions, other sections etc), while double are used for monospace. This is rather confusing when my brain is used to Markdown.

Reply:
Done.